src/etcd/client.py

```diff
@@ -42,6 +42,9 @@ class Client(object):
     _comparison_conditions = set(('prevValue', 'prevIndex', 'prevExist'))
     _read_options = set(('recursive', 'wait', 'waitIndex', 'sorted', 'quorum'))
     _del_conditions = set(('prevValue', 'prevIndex'))
+
+    http = None
```

Comment:
The class at the moment has class properties that are just constants, this deviates from that convention.

Also, I honestly prefer to beg for forgiveness than ask for permission.

So I'd catch the exception in __del__ tbh